### PR TITLE
Inject entityFieldQuery class

### PR DESCRIPTION
**Pull request by e0ipso**
See #335.

### CODE REVIEW

File: plugins/restful/RestfulDataProviderEFQ.php [hunk 1]
```diff
@@ -333,4 +340,32 @@ public function remove($id) {
...
+    return $query;
+  }
+
+  /**
+   * Gets a EFQ object.
+   *
+   * @return \EntityFieldQuery
+   *   The object that inherits from \EntityFieldQuery.
+   */
+  protected function EFQObject() {
```
Comment (amitaibu) [inline plugins/restful/RestfulDataProviderEFQ.php:366]: `getEntityFieldQueryObject` maybe ?

File: plugins/restful/RestfulDataProviderEFQ.php [hunk 2]
```diff
@@ -333,4 +340,32 @@ public function remove($id) {
...
+    $ef_class = $this->EFQClass;
```
Comment (amitaibu) [inline plugins/restful/RestfulDataProviderEFQ.php]: $ef_class => $efq_class

**Comment by amitaibu** [comment]
Minor comment, rest looks good.

**Comment by e0ipso** [comment]
Fixed.

**Comment by e0ipso** [comment]
@amitaibu this is ready to be reviewed.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**